05.wc/wc.rb

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+option = ARGV.getopts('lwc')
```

Comment:
```suggestion
options = ARGV.getopts('lwc')
```

のように複数形にした方がハッシュであることがわかりやすいです